server/lease/lessor_test.go

```diff
@@ -208,6 +209,43 @@ func TestLessorRevoke(t *testing.T) {
 	}
 }
 
+func TestLessorGrantAttachRevokeDetach(t *testing.T) {
+	lg := zap.NewNop()
+	dir, be := NewTestBackend(t)
+	defer os.RemoveAll(dir)
```

Comment:
nit: this is unnecessary, because the data directory is created via `t.TempDir()`, and it will be cleanuped automatically when the test finishes.